src/components/Auth/Footer/Footer.tsx

```diff
@@ -0,0 +1,84 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import { FilterType } from '../../../types/FilterBy';
+import { Todo } from '../../../types/Todo';
+
+type Props = {
+  filterTypes: (arg: FilterType) => void;
+  filterType: FilterType,
+  todos: Todo[],
+  deleteCompleted: () => void,
+
+};
+
+export const Footer: React.FC<Props> = ({
+  filterTypes,
+  filterType,
+  todos,
+  deleteCompleted,
+}) => {
+  const notCompletedTodos = useMemo(() => (
+    todos.filter(({ completed }) => !completed)),
+  [todos]);
+
+  const todosCompletedLength = useMemo(() => (
+    todos.filter(todo => todo.completed).length),
+  [todos]);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${notCompletedTodos.length} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={classNames(
+            'filter__link',
+            { selected: filterType },
```

Comment:
```suggestion
            { selected: filterType === FilterType.All },
```
Only one link should be active